src/components/tabs/ComponentsTab.vue

```diff
@@ -42,19 +55,43 @@ import { Component, Vue } from 'vue-property-decorator';
 import ExpandablePanel from '@/components/ExpandablePanel.vue';
 import AddNodeButton from '@/components/buttons/AddNodeButton.vue';
 import ButtonGrid from '@/components/buttons/ButtonGrid.vue';
-import { mapGetters } from 'vuex';
-import { OverviewNodes } from '@/nodes/overview/Types';
+import { OverviewCategories, OverviewNodes } from '@/nodes/overview/Types';
+import SearchBar from '@/SearchBar.vue';
+import { Getter } from 'vuex-class';
+import { EditorModel } from '@/store/editors/types';
 
 @Component({
   components: {
+    SearchBar,
     ExpandablePanel,
     AddNodeButton,
     ButtonGrid,
   },
-  computed: mapGetters(['modelEditors', 'dataEditors']),
 })
 export default class ComponentsTab extends Vue {
-  private overviewNodes = OverviewNodes;
+  private readonly overviewNodes = OverviewNodes;
+  private readonly overviewCategories = OverviewCategories;
+  private searchString = '';
+  @Getter('modelEditors') modelEditors!: EditorModel[];
+  @Getter('dataEditors') dataEditors!: EditorModel[];
+
+  private search(search: string) {
+    this.searchString = search;
+  }
+
+  private get renderedModelEditors() {
+    return this.modelEditors
+      .filter((editor) => editor.name.toLowerCase().includes(this.searchString.toLowerCase()));
+  }
+
+  private get renderedDataEditors() {
+    return this.dataEditors
+      .filter((editor) => this.shouldRender(editor.name));
+  }
```

Comment:
What do these functions do?

Reply:
They calculate which model editors and date editors should be rendered, given the search string

Reply:
Although, Idk why I didn't use the function for the first one

Reply:
`shouldRender` I mean

Reply:
I think `renderedDataEditors` may be incorrect

Reply:
Why?

Reply:
You pass in `editor.name` and not `searchString`

Reply:
Yeah, because the method `shouldRender` uses `this.searchString`. If I passed in `searchString` it would always be true, since `searchString` always contains `searchString`.